explorer/covidExplorer/CovidParams.ts

```diff
@@ -436,23 +465,17 @@ export const makeColumnDefFromParams = (
     const def = {
         ...defTemplates[metricName], // do not modify the templates
     }
+
     def.slug = buildColumnSlugFromParams(
         metricName,
         perCapitaAdjustment,
         interval,
         smoothing
     )
 
-    const perCapitaMessages: { [index: number]: string } = {
-        1: "",
-        1e3: " per thousand people",
-        1e6: " per million people",
-    }
-
     const display = def.display || {}
-    display.name = `${params.intervalTitle} ${def.name ?? display.name}${
-        perCapitaMessages[perCapitaAdjustment]
-    }`
+    display.name = columnDisplayName(params)
+    display.unit = def.display?.unit ?? ""
```

Comment:
is this line necessary?

Reply:
I could drop the null check at the end but yes we need the unit for the DataTable.